main.py

```
@@ -135,26 +238,102 @@ def profiled_code():
+        if len(self.empty_block_index) < 1:
+            return -1
+        block_index = self.empty_block_index.pop(0)
+        self.update_single_block(block_index)
+
+    def init_regular_spin(self):
+        # create a new round
+        if self.ONGOING_ROUND:
+            # msg box alert: tell user must wait until current round over
+            messagebox.showinfo("警告", "禁止同時進行多個獎勵購買.")
+            return -1
+        self.round_result_label.config(text=f"Round result:\n\n")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)